lotusrb.gemspec

```diff
@@ -28,8 +28,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'dotenv',           '~> 2.0'
   spec.add_dependency 'thor',             '~> 0.19'
 
-  spec.add_development_dependency 'bundler',   '~> 1.6'
-  spec.add_development_dependency 'rake',      '~> 10'
-  spec.add_development_dependency 'minitest',  '~> 5'
-  spec.add_development_dependency 'rack-test', '~> 0.6'
+  spec.add_development_dependency 'bundler',            '~> 1.6'
+  spec.add_development_dependency 'rake',               '~> 10'
+  spec.add_development_dependency 'minitest',           '~> 5'
+  spec.add_development_dependency 'minispec-metadata',  '~> 3.2.0'
```

Comment:
Sure!

Note: Personally I don't like to vertically align lines like this, it tends to make git diffs dirty. Is it a project guideline?